### PR TITLE
Added an data-collection notebook

### LINKED ISSUE #58: Add an inference notebook for the data being stored in the opf-datacatalog bucket
Add an inference notebook for the meetup attendees data being stored in the opf-datacatalog bucket.

### PR DESCRIPTION
**Pull request by suppathak**
Added an inference notebook which would provide us a sample instruction for fetching meetup data from the opf-datacatlog bucket and description of the data. 

closes #58

### CODE REVIEW

File: notebooks/inference_notebook.ipynb [hunk 1]
```diff
@@ -0,0 +1,287 @@
+{
```
Comment (MichaelClifford) [inline notebooks/inference_notebook.ipynb:1]: <p>Can you remove the extra column "Unnamed"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/operate-first-data-science-community/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (MichaelClifford) [inline notebooks/inference_notebook.ipynb:1]: <p>Why is this notebook called "inference_notebook"? and not "data_collection" or something like that? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/operate-first-data-science-community/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (suppathak) [reply notebooks/inference_notebook.ipynb:1]: Yes, data-collection makes more sense. May be word `inference` would create a confusion. I will change the name now. Thanks.

File: notebooks/data-collection.ipynb [hunk 2]
```diff
@@ -0,0 +1,262 @@
+{
```
Comment (oindrillac) [inline notebooks/data-collection.ipynb:1]: <p>Can we add a title/headline for this notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/operate-first-data-science-community/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (oindrillac) [inline notebooks/data-collection.ipynb:1]: <p>data-&gt;date</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/operate-first-data-science-community/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

**Review by oindrillac — COMMENTED** [review]
some minor comments otherwise lgtm 👍

**Review by oindrillac — APPROVED** [review]
/lgtm

**Pull request merged**